cdhweb/blog/views.py

```diff
@@ -60,3 +61,12 @@ class AtomBlogPostFeed(RssBlogPostFeed):
 
     feed_type = Atom1Feed
     subtitle = RssBlogPostFeed.description
+
+class BlogPostRedirectView(DetailView):
```

Comment:
We might wanna use a [RedirectView](https://docs.djangoproject.com/en/5.1/ref/class-based-views/base/#redirectview) instead, then these two functions basically get smooshed into one